staking_deposit/credentials.py

```diff
@@ -125,8 +125,8 @@ def deposit_message(self) -> DepositMessage:
     def signed_deposit(self) -> DepositData:
         domain = compute_deposit_domain(fork_version=self.chain_setting.GENESIS_FORK_VERSION)
         signing_root = compute_signing_root(self.deposit_message, domain)
-        signed_deposit = DepositData(
-            **self.deposit_message.as_dict(),
+        signed_deposit = DepositData( # type: ignore[no-untyped-call]
```

Comment:
This ignore is not needed.

Reply:
Removing it seems to make mypy pass but I think it should be kept since both the DepositData call and the .as_dict call type checks below need to be ignored. So nothing to be done here.

Reply:
ack